sql/gnomex_db_upgrade_to_5.36.0.0.sql

```diff
@@ -0,0 +1,5 @@
+use gnomex;
+
+alter table Sample add column libPrepPerformedByID int(10) null;
```

Comment:
Should this be a FK to the AppUser table? Should it be named "idLibPrepPerformedBy" (in both the DB and code) to be consistent with other FK field names?